perun-openapi/openapi.yml

```
@@ -17395,6 +17426,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /urlinjsonout/servicesManager/forceServicePropagationBulk:
+    post:
+      tags:
+        - ServicesManager
+      operationId: forceServicePropagationBulk
+      summary: Forces services propagation on defined facility.
```

If we want to have it as a single method with an optional facilityId parameter, I would at least prefer to define it in summary, e.g. : 
```suggestion
      summary: Forces services propagation on all facilities, where the services are defined, or just on the given facility.
```
or something like that.